File: python/solutions/part2/longest_word_finder.py
def find_longest_word(sentence_string):
  longest_word = ''

  words = sentence_string.split()
  for word in words:
    if len(word) > len(longest_word):
      longest_word = word
    
  return longest_word

sentence_string = "This is a long sentence with words of multiple lengths for inputting into your function"
longest_word = find_longest_word(sentence_string)
print 'Longest word is {} at a length of {}'.format(longest_word, len(longest_word))

File: javascript/solutions/part2/LongestWordFinder.js
function findLongestWord(sentenceString) {
  var longestWord = '';

  //note that this is imperfect for punctuation
  words = sentenceString.split(" ");
  for (var i = 0; i < words.length; i++) {
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  }

  return longestWord;
}

var sentenceString = "This is a long sentence with words of multiple lengths for inputting into your function.";
var longestWord = findLongestWord(sentenceString);
console.log('Longest word is ' + longestWord + ' at a length of ' + longestWord.length);
